### PR TITLE
Allowing flexible index and error names

### LINKED ISSUE #28: Ability to override IndexDocument and ErrorDocument
for a project I'm working on we discovered it'd be desirable to override the indexDocument and errorDocument - and would be great to be able to specify the desired override in the serverless.yml/serverless.js configuration, eg, something like: 

```
# serverless.yml

name: my-website
stage: dev

myWebsite:
  component: "@serverless/website"
  inputs:
    code:
      root: ./ # The root folder of your website project.  Defaults to current working directory
      src: ./src # The folder to be uploaded containing your built artifact
      hook: npm run build # A hook to build/test/do anything to your code before uploading
    region: us-east-1 # The AWS region to deploy your website into
    bucketName: myBucket # (Optional) The Bucket name where `src` files/folder will be upload. 
                         # If not provided, it will create random bucket name and upload `src` files
    env: # Environment variables to include in a 'env.js' file with your uploaded code.
      API_URL: https://api.com

    ### NEW PROPERTIES HERE
    indexDocument: index.html # (Optional) the default page for your website
    errorDocument: index.html # (Optional) the error page for your website.
    ### END NEW PROPERTIES


    # You can specify a custom domain name for your website.
    # You must have a public hosted zone available for this domain in AWS Route53.
    # This is done automatically for you if you've purchased the domain via AWS Route53.
    domain: www.example.com
```
Any chance I could submit a patch to this project to enable this functionality?

### PR DESCRIPTION
**Pull request by uxtx**
This pull request enables users to be able to override default names for index and error files for a bucket, if they so desire.

Closes #28

### CODE REVIEW

File: serverless.js [hunk 1]
```diff
@@ -56,15 +61,16 @@ class Website extends Component {
     const s3 = new aws.S3({ region: inputs.region, credentials: this.context.credentials.aws })
 
     this.context.debug(`Configuring bucket ${inputs.bucketName} for website hosting.`)
-    await configureBucketForHosting(s3, inputs.bucketName)
+
+    await configureBucketForHosting(s3, inputs.bucketName, inputs.indexDocument, inputs.errorDocument)
```
Comment (uxtx) [inline serverless.js]: I had considered modifying `configureBucketForHosting` to pass a white-listed object with configurable values; but opted to just add some positional arguments  with these overrides.

**Comment by uxtx** [comment]
Seeing this error from tests - where are env credentials defined? I don't believe I modified them...
```
> 31 |     throw new Error('Unable to run tests. AWS credentials not found in the envionrment')
```

**Comment by eahefnawy** [comment]
@uxtx don't worry about the tests. We still need to figure this out. One question though, do we not need to do something similar on the distribution?

**Comment by eahefnawy** [comment]
Thanks @uxtx ... could you add these new inputs to the readme example next to the others?

**Comment by uxtx** [comment]
@eahefnawy Sure thing! Updated!

**Pull request merged**